dist/api_data.py

```diff
@@ -1079,6 +1079,11 @@ def __cmp__(self, other):
     Config('config', '', r'''
         configuration string passed to the entry point of the
         extension as its WT_CONFIG_ARG argument'''),
+    Config('early_load', 'false', r'''
+        whether this extension should be loaded at the beginning of
+		::wiredtiger_open. Only applicable to extensions loaded via the
+		wiredtiger_open configurations string''',
+		type='boolean'),
```

Comment:
@agorrod, don't we know which extensions need to be loaded early, isn't there just the one?